internal/p2p/router.go

```diff
@@ -21,6 +21,8 @@ import (
 
 const queueBufferDefault = 32
 
+const dialRandomizerIntervalMilliseconds = 3000
```

Comment:
Maybe:
```suggestion
const dialRandomizerInterval = 3000*time.Millisecond
```

?

Though I'm not seeing where this is used at the moment.